src/pages/LandingPage.jsx

```diff
@@ -0,0 +1,8 @@
+import OurVision from "../components/ourVision";
+import timelineData from "../data/timelineData.json";
+
+const LandingPage = () => {
+  return <OurVision dataList={timelineData.timeline} />;
```

Comment:
Let's wrap this in a fragment, there'll be more components to be listed here in the future ne